mc2/controllers/docker/templates/docker/inclusion_tags/controller_object.html

```diff
@@ -9,8 +9,29 @@
       <h3 class="widget-user-username">{{controller.name}}</h3>
       <div class="widget-user-desc">{{controller.description}}</div>
       <div class="widget-user-desc">
-        {{controller.app_id }}
-        <span class="label bg-{% if controller.state == 'done' %}green{% else %}orange{% endif %}">{{controller.state}}</span>
+          {% with controller.get_status as status %}
```

Comment:
Yes. My comments in slack were general rather than specific to this PR. I don't really want to refactor all the display code here. :-)